webrender/src/render_backend.rs

```diff
@@ -484,6 +479,27 @@ impl RenderBackend {
         }
     }
 
+    fn query_document(
+        &mut self,
+        document_id: DocumentId,
+        message: DocumentMsg,
+    ) {
+        let doc = self.documents.get_mut(&document_id).unwrap();
```

Comment:
I think we'd end up with cleaner code if we fully inline this function (no need for separate function here)